src/data/ff/ff.cpp

```diff
@@ -150,14 +149,14 @@ bool Forcefield::createNETADefinitions()
 }
 
 // Return named short-range parameters (if they exist)
-const OptionalReferenceWrapper<const ForcefieldParameters> Forcefield::shortRangeParameters(std::string_view name) const
+std::optional<std::vector<double>> Forcefield::shortRangeParameters(std::string_view name) const
```

Comment:
```suggestion
const OptionalReferenceWrapper<std::vector<double>> Forcefield::shortRangeParameters(std::string_view name) const
```

With all of the places were I've passed by reference instead of value, I'm not sure if this might not turn into a temporary somewhere and throw a fit because it can't be passed by reference.  This solves that issue, but then places the limit that the parameter vector must be const.  If we're changing that vector somewhere (that I haven't found yet), then we shouldn't do any of the reference changes that I've suggested.